src/blog.atom.njk

```diff
@@ -20,6 +20,11 @@ excludeFromSitemap: true
     <link href="{{ absolutePostUrl }}"/>
     <updated>{{ (post.data.updated or post.date) | rssDate }}</updated>
     <id>{{ absolutePostUrl }}</id>
+    {%- for tag in post.data.tags %}
+      {%- if not 'io' in tag and not 'Node.js' in tag %}
+        <category term="{{ tag | slug }}" />
```

Comment:
WDYT about adding `label="…"` with the human-readable version of the tag? Here and in the other file.

Reply:
I like it. Done in https://github.com/v8/v8.dev/pull/697/commits/55f6a42d4744ca3e83816b4db1c221320b734bf0.